### DISCUSSION TITLE
Grasscutter | v1.7.4

### DISCUSSION THREAD

**Discussion by KingRainbow44** [Melledy/Grasscutter#2447] [Announcements]
# Changelog
most likely the last update for a while; the game hasn't had any recent innovations & protocol defs are harder to come by
- more spiral abyss fixes
- finally fixing the year-old bug of elemental energy being taken away without activating an avatar's burst
- bug fixes related to the player's home realm
- bug fixes with enemies (specifically substates)

## What's Changed
* Implement abyss defense objective by @longfruit in https://github.com/Grasscutters/Grasscutter/pull/2422
* Only deduct energy when elemental burst actually fires by @longfruit in https://github.com/Grasscutters/Grasscutter/pull/2424
* Fix tower mob level and hp scaling by @longfruit in https://github.com/Grasscutters/Grasscutter/pull/2430
* Fix a typo from "culivation" to "cultivation" in readme EN, zh-CN, zh-TW by @EnderNon in https://github.com/Grasscutters/Grasscutter/pull/2431
* fix: NPE related to teapot when player logs in. by @hamusuke0323 in https://github.com/Grasscutters/Grasscutter/pull/2429
* Handle mob summon and limbo state by @longfruit in https://github.com/Grasscutters/Grasscutter/pull/2432
* Update README_ja-JP.md by @Ign1s-Reiga in https://github.com/Grasscutters/Grasscutter/pull/2438

## New Contributors
* @EnderNon made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/2431

**Full Changelog**: https://github.com/Grasscutters/Grasscutter/compare/v1.7.3...v1.7.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Grasscutters/Grasscutter/releases/tag/v1.7.4'>Grasscutter | v1.7.4</a>.</em>

**Comment by anaononononanonnnnn** [comment]
absolutely useless

**Reply by EnderNon** [reply]
hi xeon 🫃
